apps/src/templates/instructions/InstructionsCsfMiddleCol.jsx

```diff
@@ -180,6 +180,7 @@ class InstructionsCsfMiddleCol extends React.Component {
             <InlineHint
               key={hint.hintId}
               borderColor={color.light_info_500}
+              backgroundColor={'#f0ffff'}
```

Comment:
also nit for these three: should we use color constants here?

Reply:
or at least put these colors in the variables with some descriptive name (store them in this file only)

Reply:
Thanks so much Denys! I will address these changes using constants in a fast follow-up. I am currently blockly @katiejofr from making a screencast video which includes these updates so will merge this PR including change @bencodeorg mentioned in his comment.

Reply:
https://trello.com/c/gTzUI8Kd/182-follow-up-to-instructions-update-cleanup-refactor-of-color-constants